### DISCUSSION TITLE
Create a pdf file with the approver's signature or seal and save that pdf file to the database

### DISCUSSION THREAD

**Discussion by QuangLanh** [clientIO/joint#2412] [Q&A]
### Introduction

Currently, we have successfully saved the drawn diagram to the database. So now, we want to do one more approval step. If the approver approves the diagram, it will be saved to the database as a pdf file along with the approver's signature or seal.

### Steps to reproduce

![image](https://github.com/clientIO/joint/assets/91791283/0c885dc2-12aa-419e-b9e2-4ffc9cb448c3)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
You can add an image to the paper that looks like a stamp or how exactly do you expect our library to help you with this task? Isn't this a task for the PDF library?

**Reply by QuangLanh** [reply]
I apologize for my confusion.
